### PR TITLE
[AIRFLOW-XXX] Fix minor typo in slack hook

### PR DESCRIPTION
**Pull request by h5chauhan**
Fixes a minor typo in the slack hook and add Acorns as user

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,6 +124,7 @@ Currently **officially** using Airflow:
 1. [8fit](https://8fit.com/) [[@nicor88](https://github.com/nicor88), [@frnzska](https://github.com/frnzska)]
 1. [90 Seconds](https://90seconds.tv/) [[@aaronmak](https://github.com/aaronmak)]
 1. [99](https://99taxis.com) [[@fbenevides](https://github.com/fbenevides), [@gustavoamigo](https://github.com/gustavoamigo) & [@mmmaia](https://github.com/mmmaia)]
+1. [Acorns Grow](https://www.acorns.com/) [[Acorns](https://github.com/Acornsgrow)]
```
Comment (mik-laj) [inline README.md]: Can you do it in a separate PR? Otherwise, this change will be difficult to cherry-pick to 1.10.X.

Comment (hsc-afk) [reply README.md]: Removed from this PR

File: airflow/hooks/slack_hook.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def __get_token(self, token, slack_conn_id):
 
     def call(self, method: str, api_params: dict) -> None:
         """
-        Calls the clack client.
+        Calls the slack client.
```
Comment (mik-laj) [inline airflow/hooks/slack_hook.py]:
```suggestion
        Calls the Slack client.
```

Comment (mik-laj) [reply airflow/hooks/slack_hook.py]: This you can commit using the Github UI.

**Comment by mik-laj** [comment]
Thanks. Awesome I merge it.

**Review by mik-laj — APPROVED** [review]
